src/controllers/routes.ts

```diff
@@ -34,6 +35,7 @@ export async function setupRouter(_: GlobalContext): Promise<Router<GlobalContex
   router.get('/users/:address/third-party-wearables/:collectionId', thirdPartyCollectionWearablesHandler)
   router.get('/third-party-integrations', thirdPartyIntegrationsHandler)
   router.get('/users/:address/emotes', emotesHandler)
+  router.get('/users/names/:name/owner', nameOwnerHandler)
```

Comment:
As you're not accessing the resource `/users`, what do you think about renaming the endpoint to:
```suggestion
  router.get('/names/:name/owner', nameOwnerHandler)
```